alarm_trap_sender.hpp

```diff
@@ -39,9 +39,23 @@
 
 #include <string>
 #include <map>
+#include <cstdint>
 
 #include "alarm_table_defs.hpp"
 
+typedef struct SNMPDateTime {
```

Comment:
It would be good to have a comment referencing the RFC - particularly to make it clear to anyone modifying this code in future that this isn't an arbitrary format, it's carefully written to match precise byte layout requirements.